electron/main/index.ts

```diff
@@ -216,6 +216,32 @@ ipcMain.handle("show-context-menu-file-item", (event, file) => {
   }
 });
 
+ipcMain.handle("show-context-menu-item", (event) => {
```

Comment:
We already have the handler for ```show-context-menu-file-item```. Why not include these new context menu items there? 

(For my personal vault, I have the fileexplorer sidebar so i could not actually get it to show these new context menu items you've added. Only by opening a vault which doesn't have lots of files would it actually leave space in the file explorer sidebar to hit these context menu items)

Reply:
Because this one is specifically for right clicking on the file explorer sidebar (if a file does not exist there) that creates a menu to create a new note or directory. The current `show-context-menu-file-item` has options like `delete`, `rename`, and `create a flashcard set` that would not be appropriate. 

This is just something I like to have for note apps, but I understand if other people do not think it's necessary!